dashboard/app/models/script.rb

```diff
@@ -998,14 +1000,31 @@ def self.add_script(options, raw_stages, new_suffix: nil, editor_experiment: nil
       end
       # Set/create Stage containing custom ScriptLevel
       if stage_name
+        # check if that lesson_group exists for the script
```

Comment:
Can you expand this comment to discuss all three cases? I like to err on the verbose side for large functions with a lot of different steps. (Potentially it would be even better to break up into smaller functions, but comments go a long way.)